packages/configs/eslint-common.json

```diff
@@ -3,9 +3,10 @@
     "extends": [
         "plugin:prettier/recommended",
         "plugin:security/recommended",
-        "./eslint-ts-overrides.json"
+        "./eslint-ts-overrides.json",
+        "plugin:no-unsanitized/DOM"
```

Comment:
Optional:

It shouldn't matter in this case, but probably external resources extension should happen before our local file.

Reply:
Fixed